CHANGES.rst

```diff
@@ -295,6 +295,8 @@ astropy.stats
 astropy.table
 ^^^^^^^^^^^^^
 
+- Allowed Columns to appear in the parameter 'tables' passed to hstack table operation.[#6791]
```

Comment:
Always use double backquotes in changelog; that is, ``tables``. Also, the number at the end denotes the PR number NOT the issue it solves; use [#6818]. Also, a space after ``.`` (trivial, but to keep with the format of the file).

Probably best to write: 
Allowed ``Column`` to be passed as valid input to the ``hstack`` Table operation. [#6818]